system-design/orchestration/workload/workload-orchestration-edge-onboarding.md

```diff
@@ -0,0 +1,70 @@
+# Device Management Client Onboarding
+In order for the workload orchestration solution to manage the edge device's workloads, the device's management client must first complete onboarding.
+
+> Action: The details in this page are still under discussion and have not been finalized.
+
+**The onboarding process includes:**
+
+1. The end user provides the the workload orchestration web service's root URL to the device's management client
+1. The device's management client downloads the workload orchestration solution vendor's public root CA certificate using the [Onboarding API](../../margo-api-reference/workload-api/onboarding-api/rootca-download.md)
+1. Context and trust is established between the device's management client and the workload orchestration web service
+1. The device's management client uses the [Onboarding API](../../margo-api-reference/workload-api/onboarding-api/device-onboarding.md) to onboard with the workload orchestration solution
```

Comment:
The concept "solution" has not been introduced before. Are you having something similar in mind as 'symphony' defines?

Reply:
changed wording to service. Think this resolved this comment.